lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -368,7 +373,7 @@ def configAPN(self, imsi, apn_list, default=True, static_ip=None):
                 apn_list.insert(0, magma_default_apn)
             else:
                 apn_list = [magma_default_apn]
-        self._sub_util.config_apn_data(imsi, apn_list)
+        # self._sub_util.config_apn_data(imsi, apn_list)
```

Comment:
Since the invoking of configAPN is guarded with flag, cloud. Do we need to comment this line?